core/src/main/java/org/geysermc/geyser/entity/type/DisplayBaseEntity.java

```diff
@@ -0,0 +1,84 @@
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * @author GeyserMC
+ * @link https://github.com/GeyserMC/Geyser
+ */
+
+package org.geysermc.geyser.entity.type;
+
+import it.unimi.dsi.fastutil.objects.ObjectArrayList;
+import net.kyori.adventure.text.Component;
+import org.cloudburstmc.math.vector.Vector3f;
+import org.cloudburstmc.protocol.bedrock.data.entity.EntityDataTypes;
+import org.geysermc.geyser.entity.EntityDefinition;
+import org.geysermc.geyser.session.GeyserSession;
+import org.geysermc.mcprotocollib.protocol.data.game.entity.metadata.EntityMetadata;
+import org.geysermc.mcprotocollib.protocol.data.game.entity.metadata.type.BooleanEntityMetadata;
+import org.jetbrains.annotations.Async;
+
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.geysermc.geyser.util.EntityUtils.getMountOffset;
+
+// Note: 1.19.4 requires that the billboard is set to something in order to show, on Java Edition
+public class DisplayBaseEntity extends Entity {
+
+    public Vector3f baseTranslation;
+
+
+    public DisplayBaseEntity(GeyserSession session, int entityId, long geyserId, UUID uuid, EntityDefinition<?> definition, Vector3f position, Vector3f motion, float yaw, float pitch, float headYaw) {
+        super(session, entityId, geyserId, uuid, definition, position, motion, yaw, pitch, headYaw);
+    }
+
+    @Override
+    protected void initializeMetadata() {
+        super.initializeMetadata();
+    }
+
+    @Override
+    public void setDisplayNameVisible(BooleanEntityMetadata entityMetadata) {
+        // Don't allow the display name to be hidden - messes with our armor stand.
+        // On JE: Hiding the display name still shows the display entity text.
+    }
+
+    @Override
+    public void setDisplayName(EntityMetadata<Optional<Component>, ?> entityMetadata) {
+        // This would usually set EntityDataTypes.NAME, but we are instead using NAME for the text display.
+        // On JE: custom name does not override text display.
+    }
+
+    public void setTranslation(EntityMetadata<Vector3f, ?> translationMeta){
+        this.baseTranslation = translationMeta.getValue();
+        if(this.vehicle == null){
+            this.setRiderSeatPosition(this.baseTranslation);
+            this.moveRelative(0, this.baseTranslation.getY(), 0, yaw, pitch, headYaw, false);
+        } else {
+            this.setRiderSeatPosition(this.baseTranslation.add(getMountOffset(this, this.vehicle)));
+        }
+
+    }
+    public void setScale(EntityMetadata<Vector3f, ?> translationMeta){
+
+    }
```

Comment:
Why is this here as an empty function?

Reply:
Text displays can't be scaled with that but only glyphs, so it's basically useless, but could be needed for future implementations.

Reply:
Your defining it as a translator but then not doing anything, might as well keep it to just a null translator

Reply:
@stromsoftware this wasn't removed, so this isn't actually resolved either